### PR TITLE
GetAllUsers endpoint

### PR DESCRIPTION
**Pull request by lalagvana**
GetAllUsers endpoint with pagination and filter by full name

### CODE REVIEW

**Review by DarkFelas — APPROVED** [review]

File: src/UserService.Data/UserRepository.cs [hunk 1]
```diff
@@ -64,5 +65,19 @@ public DbUser GetUserByEmail(string userEmail)
 
             return dbUser;
         }
+
+        public IEnumerable<DbUser> GetAllUsers(int skipCount, int takeCount, string nameFilter)
```
Comment (djino97) [inline src/UserService.Data/UserRepository.cs:69]: Rename nameFilter -> userFullName

File: src/UserService.Business/GetAllUsersCommand.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+        public GetAllUsersCommand(
+            [FromServices] IUserRepository repository,
+            [FromServices] IMapper<DbUser, User> mapper)
+        {
+            this.repository = repository;
+            this.mapper = mapper;
+        }
+
+        public IEnumerable<User> Execute(int skipCount, int takeCount, string nameFilter)
```
Comment (djino97) [inline src/UserService.Business/GetAllUsersCommand.cs:25]: Rename nameFilter -> userFullName

File: src/UserService.Data/UserRepository.cs (hunk 1 above)
Comment (lalagvana) [reply src/UserService.Data/UserRepository.cs:69]: image
That's exactly a filter, not the whole full name

File: src/UserService.Business/GetAllUsersCommand.cs (hunk 2 above)
Comment (lalagvana) [reply src/UserService.Business/GetAllUsersCommand.cs:25]: https://github.com/LT-Students/DigitalOffice-UserService/pull/19#discussion_r508705091

File: src/UserService.Data/UserRepository.cs (hunk 1 above)
Comment (djino97) [reply src/UserService.Data/UserRepository.cs:69]: it is not clear what the filter is, the nameFilter does not make it clear to me what the variable means. nameFilter -> userNameFilter

Comment (lalagvana) [reply src/UserService.Data/UserRepository.cs:69]: Agreed. Fixed

**Review by djino97 — APPROVED** [review]

**Pull request merged**